jaxrs-api/pom.xml

```diff
@@ -182,6 +182,7 @@
                         </plugin>
                         <plugin>
                             <artifactId>maven-surefire-plugin</artifactId>
+                            <version>2.21.0</version>
```

Comment:
Please adopt my real solution instead, see https://github.com/spericas/jaxrs-api/pull/1.

Reply:
It's taking way too much time for a simple change. Let's get this in and then we can review your PR to improve this later.

Reply:
I will not accept a workaround getting in hastily once the actual solution is on the table -- it is away a single click, so why not doing it?